admin_guide/managing_networking.adoc

```diff
@@ -144,6 +144,39 @@ $ systemctl restart atomic-openshift-master-api atomic-openshift-master-controll
 
 ----
 
+[[admin-guide-managing-networking-ingress]]
+== Setting up the HAProxy router to accept ingress objects
+
+By default, the HAProxy router only accepts route objects. You can set up the
+HAProxy router as a Kubernetes
+link:https://kubernetes.io/docs/concepts/services-networking/ingress/[Ingress
+Controller] so that it accepts both route and ingress objects. If configured,
+the same rules apply for claiming a domain for a namespace based on the creation
+time of the object. For example, ownership of routes is based on the claims made
+for a host name with the namespace with the oldest route winning against any
+other claimants.
+
```

Comment:
This sentence seems awkward (or a run on?) because of the two _with the_ phrases. But, I see that the same sentence is in the Install Guide, so maybe I am mistaken.
https://docs.openshift.com/container-platform/3.7/install_config/router/default_haproxy_router.html#using-wildcard-routes.